velox/docs/functions/spark/string.rst

```diff
@@ -75,6 +75,14 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT find_in_set(NULL, ',123'); -- NULL
         SELECT find_in_set("abc", NULL); -- NULL
 
+.. spark:function:: initcap(str) -> string
+
+    Translate the first letter of each word to upper case in the sentence.
```

Comment:
It seems the other letters will be converted to lower-case. Could we clarify this in the documentation?